src/js/containers/homepage/CovidHighlights.jsx

```diff
@@ -308,8 +306,17 @@ export class CovidHighlights extends React.Component {
                             </span>
                         </h1>
                         <p>
```

Comment:
I'm getting console errors `Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    in div (created by CovidHighlights)` and `Warning: validateDOMNesting(...): <hr> cannot appear as a descendant of <p>.
    in hr (created by CovidHighlights)` that I think would be resolved by changing this to a `<div>`